GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -409,7 +409,12 @@ private static ArgumentString FindRenamesAndCopiesOpts()
             return FindRenamesOpt() + findCopies;
         }
 
-        public void SetPathFilters(string path)
+        public void DisableFilters()
+        {
+            _formRevisionFilter.DisableFilters();
+        }
+
+        public void SetPathFilter(string path)
```

Comment:
We don't use "disable/clear" for other filters, instead we set those to `""`. Any reason you thought it wouldn't work in this case?